File: app/src/main/java/com/zzzmode/appopsx/ui/model/PermissionGroup.java
package com.zzzmode.appopsx.ui.model;

import java.util.List;

public class PermissionGroup {

  public String group;
  public String opName;
  public String opPermsName;
  public String opPermsLab;
  public String opPermsDesc;
  public int grants;
  public int count;
  public int icon;
  public List<PermissionChildItem> apps;

  @Override
  public String toString() {
    return "PermissionGroup{" +
        "opName='" + opName + '\'' +
        ", opPermsName='" + opPermsName + '\'' +
        ", opPermsLab='" + opPermsLab + '\'' +
        ", opPermsDesc='" + opPermsDesc + '\'' +
        ", grants=" + grants +
        ", count=" + count +
        ", apps=" + apps +
        '}';
  }
}

File: app/src/main/kotlin/com/zzzmode/appopsx/ui/model/PermissionGroup.kt
package com.zzzmode.appopsx.ui.model

class PermissionGroup {

    lateinit var group: String
    var opName: String? = null
    var opPermsName: String? = null
    var opPermsLab: String? = null
    var opPermsDesc: String? = null
    var grants: Int = 0
    var count: Int = 0
    var icon: Int = 0
    var apps: MutableList<PermissionChildItem> = ArrayList()

    override fun toString(): String {
        return "PermissionGroup{" +
                "opName='" + opName + '\''.toString() +
                ", opPermsName='" + opPermsName + '\''.toString() +
                ", opPermsLab='" + opPermsLab + '\''.toString() +
                ", opPermsDesc='" + opPermsDesc + '\''.toString() +
                ", grants=" + grants +
                ", count=" + count +
                ", apps=" + apps +
                '}'.toString()
    }
}
